src/static/js/elements-identification.js

```diff
@@ -24,9 +24,10 @@ import {
   updateLoadOptions,
   ShapeUtil,
   addOverlays,
+  removeAllOverlays,
 } from '../../index.es.js';
 
-let lastBpmnIdsWithExtraCssClasses = [];
+let lastBpmnIdsWithExtraCssClassesAndOverlays = [];
```

Comment:
❓ why not calling it `lastIdentifiedBpmnIds` for simplicity?

Reply:
Good idea :)